libs/agent-scripts/cloudbroker/upgrade_cluster.py

```diff
@@ -20,17 +19,20 @@ def action():
     """
     initiate upgrade
     """
+    import yaml
     zero_complexity_path = '/opt/code/github/0-complexity/'
     repo_path = zero_complexity_path + 'openvcloud_installer'
-
-    if j.system.fs.exists(repo_path):
-        j.system.process.execute('cd %s && git reset --hard' % repo_path)
-        j.system.process.execute('cd %s && git checkout master' % repo_path)
-        j.system.process.execute('cd %s && git pull' % repo_path)
-    else:
-        j.system.process.execute(
-            'cd %s && git clone https://github.com/0-complexity/openvcloud_installer.git' % zero_complexity_path)
-
+    scl = j.clients.osis.getNamespace('system')
+    versionmodel = scl.version.searchOne({'status': 'INSTALLING'})
+    manifest = yaml.load(versionmodel['manifest'])
+    for repo in manifest['repos']:
+        if repo['url'] == 'https://github.com/0-complexity/openvcloud_installer':
+            tag = repo['target']['tag']
+            break
+
+    j.system.process.execute('kubectl delete configmap manifest-version', dieOnNonZeroExitCode=False)
```

Comment:
it doesnt make sense that the kubectl underneed needs --kubeconfig and this doesnt? Either both need it or neither right?